Content.Server/Nuke/NukeSystem.cs

```diff
@@ -406,6 +426,18 @@ private void PlayNukeKeypadSound(EntityUid uid, int number, NukeComponent? compo
                 Filter.Pvs(uid), uid, AudioHelpers.WithVariation(varyPitch).WithVolume(-5f));
         }
 
+        public string GenerateRandomNumberString(int length)
+        {
+            var ret = "";
+            for (var i = 0; i < length; i++)
+            {
+                var c = (char) _random.Next('0', '9' + 1);
+                ret += c;
+            }
+
+            return ret;
+        }
```

Comment:
🤨 

wouldn't `Next(0,Pow(10, length))` work just fine.

Reply:
copy pasted from the old nuke code system code 😎

Reply:
an actual answer: the old nuke code system code seemed to just generate a string of some length made of numbers using this, and i'm just reusing that same code to generate both the nuke code (which was also stored as a string, which is fine enough) and the serial number for nuclear bombs